policyengine_us/variables/gov/usda/csfp/csfp.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_us.model_api import *
+
+
+class csfp(Variable):
```

Comment:
```suggestion
class commodity_supplemental_food_program(Variable):
```

Reply:
I think for further analysis it would be clearer to have this written out - same with file and test name

Reply:
Do you want me to rename the directories as well?

Reply:
I think it is fine as it is